src/metrics/mod.rs

```diff
@@ -67,6 +78,64 @@ metrics! {
     namespace: "docsrs",
 }
 
+#[derive(Debug, Default)]
+pub(crate) struct RecentlyAccessedReleases {
+    crates: DashMap<i32, Instant>,
+    versions: DashMap<i32, Instant>,
+    platforms: DashMap<(i32, TargetAtom), Instant>,
+}
+
+impl RecentlyAccessedReleases {
+    pub(crate) fn new() -> Self {
+        Self::default()
+    }
+
+    pub(crate) fn record(&self, krate: i32, version: i32, target: &str) {
+        self.crates.insert(krate, Instant::now());
+        self.versions.insert(version, Instant::now());
+        self.platforms
+            .insert((version, TargetAtom::from(target)), Instant::now());
```

Comment:
Let's make these all the same instant.
```suggestion
        let now = Instant::now();
        self.crates.insert(krate, now);
        self.versions.insert(version, now);
        self.platforms.insert((version, TargetAtom::from(target)), now);
```